tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -870,6 +870,43 @@ def solve_tridiag(
     np.allclose(expected, out)
 
 
+def test_docstring():
+    size = 10
+    a = np_as_located_field(IDim)(np.ones((size,)))
+
+    @field_operator
+    def fieldop_with_docstring(a: Field[[IDim], float64]) -> Field[[IDim], float64]:
+        """My docstring."""
+        return a
+
+    @program
+    def test_docstring(a: Field[[IDim], float64]) -> Field[[IDim], float64]:
+        """My docstring."""
+        fieldop_with_docstring(a, out=a)
+
+    test_docstring(a, offset_provider={})
+
+
```

Comment:
Why is this here? Let's put it in a separate PR, if you want to add this test. Or is a merge with the main branch missing?

Reply:
there was some duplication when I updated my branch off of functional